File: javapatterns/src/main/java/org/iut/nantes/facade/cours/Ordinateur.java
package org.iut.nantes.facade.cours;

public class Ordinateur {
    public void brancherOrdi(Multiprises multiprises) {
        System.out.println("Ordinateur branché");
    }

    public void débrancherOrdi(Multiprises multiprises) {
        System.out.println("Ordinateur débranché");
    }

    public void allumerOrdinateur() {
        System.out.println("Ordinateur allumé");
    }

    public void éteindreOrdinateur() {
        System.out.println("Ordinateur éteint");
    }
}

File: kotlinpatterns/src/main/kotlin/org/iut/nantes/facade/cours/Ordinateur.kt
package org.iut.nantes.facade.cours

class Ordinateur {
    fun brancherOrdi(multiprises: Multiprises?) {
        println("Ordinateur branché")
    }

    fun débrancherOrdi(multiprises: Multiprises?) {
        println("Ordinateur débranché")
    }

    fun allumerOrdinateur() {
        println("Ordinateur allumé")
    }

    fun éteindreOrdinateur() {
        println("Ordinateur éteint")
    }
}